ludwig/schema/features/utils.py

```diff
@@ -99,6 +117,12 @@ def get_output_feature_jsonschema():
         },
     }
 
+    if model_type == MODEL_GBM:
+        prune_gbm_features(schema)
+        schema["maxItems"] = 1
```

Comment:
@justinxzhao This particular line may mean you can delete the extra validation check you're writing for gbm models to have one output feature

Reply:
Cool. What would the error message from this be? If the error message is cryptic, perhaps there's an argument to keeping both around.

Reply:
@justinxzhao Good question. It would be a standard JSON validation error message that would depend on the particular validator package (and version) in use. But it would read something like
```
output_features array too long / only supports length of 1
```

Neither too cryptic nor as human readable as it could be. So yeah, might as well keep it around for now. We might consider in the future exploring a workstream where we override JSON validation error messages within Ludwig - I believe the `jsonschema` package we use has support for this.

Reply:
@justinxzhao my bad I deleted your comment :(